eq_cir_management_ui/main/routes.py

```diff
@@ -0,0 +1,39 @@
+"""Routes for the EQ CIR Management UI."""
+
+import logging
+
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+)
+
+main_blueprint = Blueprint("main", __name__)
+
+logger = logging.getLogger(__name__)
+
+
+@main_blueprint.before_request
+def before_request_func() -> None:
```

Comment:
`before_request_func` -> `before_request`

Reply:
Not sure what this means